### DISCUSSION TITLE
Project App Selection: Which app your choose?

### DISCUSSION THREAD

**Discussion by stan6** [SOEN345-WINTER2024/discussion#4] [Q&A]
Which app your choose? (Use the GitHub repository name: owncloud/android).

Example answer:
Group Name:  MyGroup
Selected App1:  cloud1/android1 (ID: ...)
Selected App2: cloud2/android2  (ID: ...)
Selected App3: cloud3/android3  (ID: ...)

**Comment by mzmddi** [comment]
Group Name : Testers
Selected App 1 : [tasks/tasks](https://github.com/tasks/tasks) (Muiz Madadi ID: 40226708)  
Selected App 2 : [hieuwu/android-groceries-store](https://github.com/hieuwu/android-groceries-store) (Tanya So Tin Yan ID : 40208954)
Selected App 3 : [lucasnlm/antimine-android](https://github.com/lucasnlm/antimine-android) (Asmae Loulidi 40210936)
Selected App 4 : [iSoron/uhabits](https://github.com/iSoron/uhabits) (Fadoua Doghmane ID : 40198495)

**Comment by laurenrigante** [comment]
Group Name: BugBusters

Selected App 1: [lucasnlm/antimine-android](https://github.com/lucasnlm/antimine-android) (Lauren Rigante: 40188593)
Selected App 2: [K-9 mail thunderbird/thunderbird-android](https://github.com/thunderbird/thunderbird-android) (Layana  Muhdi Al-Tounsi : 40125569)
Selected App 3: [AntennaPod/AntennaPod](https://github.com/AntennaPod/AntennaPod) (Samuel Collette: 40175048)

**Comment by KhalilGarali** [comment]
Group Name: KWY
Selected App1: [beemdevelopment/Aegis](https://github.com/beemdevelopment/Aegis) (Khalil Garaali 40226310)  
Selected App 2: [K-9 mail thunderbird/thunderbird-android](https://github.com/thunderbird/thunderbird-android) (Wadeh Hamati : 40216893)

**Comment by daria-koroleva** [comment]
Group Name: pixelcure

Selected App 1: [AntennaPod/AntennaPod](https://github.com/AntennaPod/AntennaPod?tab=readme-ov-file) (Daria Koroleva 40096402)
Selected App 2: [simplenote-android](https://github.com/Automattic/simplenote-android) (Fatoumata Binta Barry 40213443)
Selected App3: [federicoiosue/Omni-Notes](https://github.com/federicoiosue/Omni-Notes.git) (Ihana Fahmy - 40209686)
Selected App4: [game2048](https://github.com/andstatus/game2048) (Shamma Markis - 40211998)

**Comment by Zakaria0907** [comment]
Group Name: Gatorade

Selected App 1: [federicoiosue/Omni-Notes](https://github.com/federicoiosue/Omni-Notes.git) (Zakaria El Manar El Bouanani 40190432)
Selected App 2: [sadellie/unitto](https://github.com/sadellie/unitto) (Anes Khadiri 40159080)
Selected App 3: [spacecowboy/NotePad](https://github.com/spacecowboy/NotePad/tree/master) (Abdelkader Habel 40209153)
Selected App 4: [gokadzev/Musify](https://github.com/gokadzev/Musify) (Omar Fares 40162541)

**Comment by NicolasChelico** [comment]
Group Name: Team Red Dragon
Selected App 1:  https://github.com/Automattic/simplenote-android (Nicolas Chelico 40156158)
Selected App 2: https://github.com/SimpleMobileTools/Simple-Gallery (Charles Eimer 26747310)
Selected App 3: https://github.com/Nain57/Smart-AutoClicker (Dan Duguay 40202775)

**Reply by DanDuguay** [reply]
Selected App 3: [Smart-AutoClicker](https://github.com/Nain57/Smart-AutoClicker) (Daniel Duguay 40202775)

**Comment by TuanAnh-P** [comment]
Group Name: 6ix
Selected App 1: [wordpress-mobile/WordPress-Android](https://github.com/wordpress-mobile/WordPress-Android) (Tuan Anh Pham : 40213926)
Selected App 2: [TeamAmaze/AmazeFileManager](https://github.com/TeamAmaze/AmazeFileManager) (Ozan Alptekin : 40162877)
Selected App 3: [mozilla-mobile/firefox-android](https://github.com/mozilla-mobile/firefox-android) (Mustafa Sameem : 40190889)

**Reply by ozan1318** [reply]
Selected App 2: https://github.com/TeamAmaze/AmazeFileManager (Ozan Alptekin 40162877)

**Reply by MustafaSameem** [reply]
Selected App 3: https://github.com/mozilla-mobile/firefox-android (Mustafa Sameem 40190889)

**Comment by RyanGuzelian** [comment]
Group Name: BugBreakers
Selected App 1: [KotatsuApp/Kotatsu](https://github.com/KotatsuApp/Kotatsu) (Ryan Guzelian 40211846)
Selected App 2: [SimpleMobileTools/Simple-Gallery](https://github.com/SimpleMobileTools/Simple-Gallery) (Carla Shawi 40190561)
Selected App 3: [cyclestreets/android](https://github.com/cyclestreets/android) (Tharsipa Nadarajah - 40190597)

**Comment by yasminehilout** [comment]
Group Name:  \`${name}\`
Selected App1: https://github.com/jellyfin/jellyfin-androidtv (Yasmine Hilout 40214158)
Selected App2:  https://github.com/duckduckgo/android (Gabriel Dubois 40209252)
Selected App3:  https://github.com/Swati4star/Images-to-PDF (Ali Fetanat 40158208)

**Comment by Horisofine** [comment]
Group Name: Big Mac Trio
Selected App1: [tasks/tasks](https://github.com/tasks/tasks) (Houssam Eddine Righi 40155074)
Selected App2: [jellyfin](https://github.com/jellyfin/jellyfin-androidtv) (Ceyhun Topcu 40159200)
Selected App3: [a-flash-deck](https://github.com/rh-id/a-flash-deck) (Mohammed Rahman 40203098)

**Reply by Afifr2001** [reply]
Selected App3: https://github.com/rh-id/a-flash-deck (Mohammed Rahman 40203098)

**Comment by gevorgalaverdyan** [comment]
Group Name: PewPew
Selected App1: [TeamNewPipe/NewPipe](https://github.com/TeamNewPipe/NewPipe) (Gevorg Alaverdyan #40202177 )
Selected App2: [iSoron/uhabits](https://github.com/iSoron/uhabits/tree/dev) (Khashayar Azad #40211574)
Selected App3: [konradrenner/kolabnotes-android](https://github.com/konradrenner/kolabnotes-android) (Yeprem Antranik #40204291)
Selected App4: [QuantumBadger/RedReader](https://github.com/QuantumBadger/RedReader) (Jay Patel #40203705)

**Comment by ramy-att** [comment]
Group Name: Muffin

Selected App 1: https://github.com/QuantumBadger/RedReader (Ramy Attalla #40205642)
Selected App 2: https://github.com/yuliskov/SmartTube (Tarek Elalfi #40197527)
Selected App 3: https://github.com/beemdevelopment/Aegis (Youssef Morcos #40124441)

**Comment by v-vasco** [comment]
Group Name: Droid Squad

Selected App 1: [/Anuken/Mindustry](https://github.com/Anuken/Mindustry) (Victor Vasconcelos 40177135)
Selected App 2:  [/Darkempire78/OpenCalc](https://github.com/Darkempire78/OpenCalc) (Carter Stroeder 40121935)
Selected App 3:  [/moneymanagerex/android-money-manager-ex](https://github.com/moneymanagerex/android-money-manager-ex) (Jean-Claude Abou-Elias 40086851)
Selected App 4:   [/gsantner/markor/](https://github.com/gsantner/markor/) (Hassan Moharram 40158285)

**Reply by DobDub** [reply]
https://github.com/gsantner/markor.git (Hassan Moharram 40158285)

**Reply by jcjc1233** [reply]
Selected App 3: (Jean-Claude Abou-Elias 40086851) https://github.com/moneymanagerex/android-money-manager-ex

**Comment by sonalipxtel** [comment]
Group Name: TestingApps

Selected App 1: [lexica/lexica](https://github.com/lexica/lexica) (Sonali Patel 40176580)
Selected App 2: [Neamar/KISS](https://github.com/Neamar/KISS) (Rana Gohar 40117475)
Selected App 3: [grote/Transportr](https://github.com/grote/Transportr) (Vanisha Patel 40242554)
Selected App 4: [Anuken/Mindustry/tree/master](https://github.com/Anuken/Mindustry/tree/master) (Marina Girgis 40168639)

**Reply by khaled-saleh1** [reply]
Hello I just want to let you know that the New Pipe app is already chosen twice. I checked before adding mine and it was only picked once by Group: PewPew. Just wanted to let you know!

**Comment by khaled-saleh1** [comment]
Group Name: OnlyTest

Selected App 1: https://github.com/konradrenner/kolabnotes-android (Mario El Shaer 40210124)
Selected App 2: https://github.com/TeamNewPipe/NewPipe  (Khaled Saleh 40210125) 
Selected App 3:  https://github.com/spacecowboy/NotePad/tree/master (Omar Shehata 40164193))

**Comment by jesseythach** [comment]
Group Name: 3ACES

Selected App 1: [Swati4star/Images-to-PDF](https://github.com/Swati4star/Images-to-PDF) (Jessey Thach 40210440)
Selected App 2: [grote/Transportr](https://github.com/grote/Transportr) (Michaël Gugliandolo 40213419)
Selected App 3: [hieuwu/android-groceries-store](https://github.com/hieuwu/android-groceries-store) (Hoang Minh Khoi Pham 40162551)

**Comment by Spiratatoe** [comment]
Group Name: ProjectTeam!
Selected App1: https://github.com/duckduckgo/android ( Jack Spiratos 40210312 )
Selected App2: https://github.com/you-apps/ClockYou ( Renaud Senécal 40208309 )
Selected App3: https://github.com/ankidroid/Anki-Android.git ( Valeria Dolgaliova 40212218 )

**Comment by Itek01** [comment]
Group Name: shadow wizards
Selected App1: https://github.com/rh-id/a-flash-deck (Adel Bouchatta 40175598)
Selected App2: https://github.com/sadellie/unitto (Chems-Eddine Saidi 40192094)
Selected App3:

**Reply by ChemsCode** [reply]
Selected App2: https://github.com/sadellie/unitto (Chems-Eddine Saidi 40192094)

**Comment by ChiefsBestPal** [comment]
Group Name: Andruides
Selected App1: https://github.com/ankidroid/Anki-Android (Antoine Cantin 40211205)
Selected App2: https://github.com/wikimedia/apps-android-wikipedia (Ryan Li 40214839)
Selected App3: https://github.com/Neamar/KISS (Abdullah Amir 40215286)

**Reply by AA789-ai** [reply]
Selected App3: [Neamar/KISS](https://github.com/Neamar/KISS) (Abdullah Amir 40215286)

**Comment by Ziadsharkos** [comment]
Group Name: SOEN345
Selected App1: [lexica/lexica](https://github.com/lexica/lexica/) (Ziad Elsharkawi 40213438)
Selected App2: [wikimedia/apps-android-wikipedia](https://github.com/wikimedia/apps-android-wikipedia?utm_source=mybridge&utm_medium=email&utm_campaign=read_more) (Poula Farid 40208791)
Selected App3: [connectbot/connectbot](https://github.com/connectbot/connectbot/) (Walid Achlaf 40210355)
Selected App4: [CatimaLoyalty/Android](https://github.com/CatimaLoyalty/Android) (Karin Sarkis 40189273)

**Reply by KarinSarkis** [reply]
Selected App4: https://github.com/CatimaLoyalty/Android (Karin Sarkis 40189273)

**Reply by walidoow** [reply]
Selected App3: [connectbot/connectbot](https://github.com/connectbot/connectbot/) (Walid Achlaf 40210355)

**Comment by amirhossein942** [comment]
Group Name: Meowerz

Selected App 1: https://github.com/CatimaLoyalty/Android (Amirhossein Tavakkoly 40203604)
Selected App 2: https://github.com/konradrenner/kolabnotes-android (Cristian Gasparesc 40209205)
Selected App 3: https://github.com/gokadzev/Musify (Sophie Hajjar 40210405)

**Comment by huy2272** [comment]
Group Name: BADminton
Selected App1: https://github.com/hanjoongcho/aaf-easydiary (Khanh Huy Nguyen 40125396)
Selected App2: https://github.com/WrichikBasu/ShakeAlarmClock (Ralph Elhage 40131981)
Selected App3: https://github.com/Docile-Alligator/Infinity-For-Reddit (Yash Patel 40175454)
Selected App4: https://github.com/freeotp/freeotp-android (Megan Coscia 40214186)

**Reply by m-coscia** [reply]
Selected App4: https://github.com/freeotp/freeotp-android (Megan Coscia 40214186)

**Comment by AyoMahan** [comment]
Group Name : opentesters
Selected App 1 :  https://github.com/ankidroid/Anki-Android.git (Mahanaim Rubin Yo - 40178119)
Selected App 2 :  https://github.com/jellyfin/jellyfin-androidtv (Nicholas Werugia - 40131956)
Selected App 3 :  https://github.com/M66B/FairEmail (Jonah Ball - 40178421)

**Reply by NicholasWahome** [reply]
Selected App 2: https://github.com/recloudstream/cloudstream (Nicholas Werugia - 40131956) 
Deselect App 2: https://github.com/jellyfin/jellyfin-androidtv (Nicholas Werugia - 40131956)

**Reply by darealstyl** [reply]
Selected App 3: https://github.com/hidroh/materialistic (Jonah Ball - 40178421)

**Comment by L3CH3R0** [comment]
Group Name: HAHS
Selected App 1 : BinaryEye (Ishaï Cohen - 40188880)
Selected App 2 : muzei/muzei (Eden Oliel - 40211989)
Selected App 3 :

**Reply by eo2000** [reply]
Selected App 2 : [muzei/muzei](https://github.com/muzei/muzei) (Eden Oliel - 40211989)

**Reply by j3rrimmy** [reply]
Selected App 3: https://github.com/lucasnlm/antimine-android (Jeremy Rimokh - 40110746)

**Reply by JeremieGar** [reply]
Selected App 4: https://github.com/alexandregpereira/Monster-Compendium (Jeremie Garzon 40062316)

**Comment by HelloMeFriend** [comment]
Group Name: yo
Selected App1: [Kotatsu](https://github.com/KotatsuApp/Kotatsu) (ID: Omar Alshanyour #40209637)
Selected App2: cloud2/android2 (ID: Vithujanan Vigneswaran #40157822)
Selected App3: cloud3/android3 (ID: Omar Alshanyour/Vithujanan Vigneswaran #40209637/#40157822)

**Reply by MinhDuc1711** [reply]
Selected App3: https://github.com/UweTrottmann/SeriesGuide (ID: Minh Duc Vu - 40166824)

**Reply by JacksnJean** [reply]
Selected App3: https://github.com/bitcoin-wallet/bitcoin-wallet.git (Jackson Jean 23507416)

**Reply by Vithujanan** [reply]
Selected App2: https://github.com/hanjoongcho/aaf-easydiary (ID: Vithujanan Vigneswaran #40157822)

**Comment by SamyArab** [comment]
Group Name: tklachesamyalex
Selected App1: https://github.com/TeamAmaze/AmazeFileManager - (ID: 40211033 Lachezara Lazarova)
Selected App2: https://github.com/Darkempire78/OpenCalc (ID: 40209204 Alexander De Luca)
Selected App3: https://github.com/yuliskov/SmartTube (ID: 40209595 Samy Arab)
Selected App4: https://github.com/google/ExoPlayer  (ID: 40192306 Thaneekan Thankarajah)

**Reply by dlu2002** [reply]
Selected App2: https://github.com/Darkempire78/OpenCalc

**Comment by LiamDaigle** [comment]
Group Name: Plant Waterers
Selected App1: https://github.com/TeamAmaze/AmazeFileManager (Liam Daigle - ID: 40207583 )
Selected App2: https://github.com/andstatus/game2048 (Justin Cheng - ID: 40210279 )
Selected App3: https://github.com/flauschtrud/broccoli (Louis Villemure - ID:40210315 )

**Reply by justncheng** [reply]
App2: https://github.com/andstatus/game2048 (Justin Cheng)

**Reply by lo-vil** [reply]
App3: https://github.com/flauschtrud/broccoli (Louis Villemure)

**Comment by s-vali** [comment]
Group Name: S345-Project
Selected App1: [https://github.com/Docile-Alligator/Infinity-For-Reddit](https://github.com/Docile-Alligator/Infinity-For-Reddit) (Sofia Valiante - ID: 40191897)
Selected App2: [https://github.com/shadowsocks/shadowsocks-android](https://github.com/shadowsocks/shadowsocks-android) (Gulnoor Kaur - ID: 40114998)
Selected App3: [https://github.com/WrichikBasu/ShakeAlarmClock](https://github.com/WrichikBasu/ShakeAlarmClock) (Adam Boucher - ID: 40165035)

**Reply by gul2223** [reply]
https://github.com/you-apps/ClockYou

**Comment by Ashx11** [comment]
Group Name: Team404
Selected App 1 : https://github.com/gsantner/markor.git (Arshpreet Singh-ID: 40172137)
Selected App 2 : [Signal-Android](https://github.com/signalapp/Signal-Android) (Phuong Khanh Ton ID:40228962)
Selected App 3 :  https://github.com/jonasoreland/runnerup/tree/master/app (Kaothar Reda-ID: 40111879)

**Reply by KaotharReda** [reply]
Selected App 3 : https://github.com/jonasoreland/runnerup/tree/master/app (Kaothar Reda-ID: 40111879)

**Comment by Ghawi25** [comment]
Group Name: H&M
Selected App 1 : https://github.com/gsantner/markor.git (Alexander El Ghaoui - 40200062)
Selected App 2 : https://github.com/Swati4star/Images-to-PDF (Youssef Enein - 40115494)

**Comment by elion-abdyli** [comment]
Group Name: Tea
Selected App 1: [termux-app](https://github.com/termux/termux-app) (Elion Abdyli - 40132982)

**Comment by KiwamiMeansExtreme** [comment]
Group Name: team sadness

Selected app 1 https://github.com/SkyTubeTeam/SkyTube (Skytube) [Jean-Nicolas Sabatini-Ouellet, 40207926]

Selected app 2 --https://github.com/mueller-ma/TabletopTools (TableTopTools) [Daniel Bondar, 40213095]
